### DISCUSSION TITLE
Alpha 5 is released

### DISCUSSION THREAD

**Discussion by EmiyaSyahriel** [EmiyaSyahriel/CrossLauncher#19] [News & Updates]
[![YouTube thumbnail](https://img.youtube.com/vi/0Bj71yJqC2w/0.jpg)](https://youtu.be/0Bj71yJqC2w)
[Preview Video](https://youtu.be/0Bj71yJqC2w)

# You can check the changelog at the [Release Page](https://github.com/EmiyaSyahriel/CrossLauncher/releases/tag/alpha)

**Comment by PSPhoneLemming** [comment]
epic!
